core/templates/components/certificate-assessment-offering-helper/certificate-offering-details.component.html

```
@@ -0,0 +1,12 @@
+<div class="oppia-certificate-offering-details">
+  <h2>Certificate Details</h2>
+  <!--
+    Real form fields for title, description, classroom, question count,
+    time limit, and demonstrates configuration are added with backend wiring.
+  -->
+
+  <div class="oppia-certificate-offering-details-actions">
+    <button (click)="onCancelClicked()">Cancel</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)